pkg/storage/segments/repairer.go

```diff
@@ -74,14 +77,19 @@ func (repairer *Repairer) Repair(ctx context.Context, path storj.Path) (err erro
 	numHealthy := len(pieces) - len(missingPieces)
 	// irreparable piece
 	if int32(numHealthy) < pointer.Remote.Redundancy.MinReq {
+		mon.Meter("repair_nodes_unavailable").Mark(1)
 		return Error.New("piece %v cannot be repaired", path)
 	}
 
 	// repair not needed
 	if int32(numHealthy) > pointer.Remote.Redundancy.RepairThreshold {
+		mon.Meter("repair_unnecessary").Mark(1)
 		return Error.New("piece %v with %d pieces above repair threshold %d", path, numHealthy, pointer.Remote.Redundancy.RepairThreshold)
 	}
 
+	healthyRatioBeforeRepair := float64(numHealthy) / float64(pointer.Remote.Redundancy.Total)
```

Comment:
Both of these value shouldnt be a float ever right?
I feel there is a better solution for that..

Reply:
I was just casting them so that we don't do integer division